app/controllers/concerto_weather/search_controller.rb

```diff
@@ -12,9 +12,11 @@ def find_city
       def getOpenWeatherCities(query)
         require 'net/http'
         require 'json'
+        require 'erb'
```

Comment:
Apparently URI.escape is deprecated? Thats why I used the erb version. But open to suggestions. I'll change the variable names asap. 

Thanks